README.md

```diff
@@ -53,6 +53,12 @@ Start the server using:
 uvicorn tdp_server.main:app --reload
 ```
 
+## Build Docker Container
+
+```bash
+podman build -t tdp_server -f docker/Dockerfile .
```

Comment:
We use `podman` command now instead of `docker` ?

Reply:
It's just an example of how to launch it, since I use podman, I put the command with podman.

Note the :Z at the end of volume mount, that's only need to configure selinux context, on ubuntu it's not needed

Reply:
I added an example with docker